up42/tasking.py

```diff
@@ -105,5 +106,54 @@ def construct_order_parameters(
 
         return order_parameters
 
+    def _query_paginated_output(self, url: str):
```

Comment:
More of a question: so instead of pagination, the sdk will return all result directly?

Reply:
Yes, we usually handle pagination within the SDK.